libstuff/SLog.cpp

```diff
@@ -114,3 +73,7 @@ string addLogParams(string&& message, const STable& params) {
 
     return message;
 }
+
+void SWhitelistLogParams(set<string> params) {
```

Comment:
NAB: add a comment on what this does/when to use

Reply:
Done